src/main/java/service/RacingGameService.java

```diff
@@ -0,0 +1,70 @@
+package service;
+
+import exception.ErrorMessage;
+import model.Car;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+public class RacingGameService {
+
+    private final static int MAX_NAME_LENGTH = 5;
+    private final static int MIN_RANDOM_NUMBER = 0;
+    private final static int MAX_RANDOM_NUMBER = 9;
+    private final static int STANDARD_NUMBER = 4;
+
+    public List<Car> parseCarNames(String carsName){
+        String[] names = carsName.split(",",-1);
+        validationName(names);
+        return Arrays.stream(names)
+                .map(String::trim)
+                .map(Car::new)
+                .toList();
+    }
+
+    public int getRandomNumber(){
+        return Randoms.pickNumberInRange(MIN_RANDOM_NUMBER, MAX_RANDOM_NUMBER);
+    }
+
+    public void racing(List<Car> cars){
+        for (Car car : cars) {
+            int randomNum = getRandomNumber();
+            if(randomNum >= STANDARD_NUMBER){
```

Comment:
```suggestion
            if (randomNum >= STANDARD_NUMBER) {
```
코드 포맷팅을 해보면 좋을 것 같아요.
Intellij라면 `Commend + Option + L`(Mac) 혹은 `Ctrl + Alt + L`(Windows) 을 사용하면 편리합니다~